### ISSUE TITLE
Fix Warnings Raised in Tile.py

### ISSUE THREAD

**Issue by coreylammie** [coreylammie/MemTorch#26]
In line 41 & 43: `torch.tensor(new_array)` => `new_array.clone().detach()`
In line 114: `tiles = torch.tensor([np.array(tile.array) for tile in tiles])` => `tiles = torch.tensor([np.array(tile.array.cpu()) for tile in tiles])`
